db/seeds.rb

```diff
@@ -5,3 +5,48 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+
+cats = [
+  {
+    name: 'Mittens',
+    age: 5,
+    enjoys: 'sunshine and warm spots',
+    profile_pic:
+      'https://images.unsplash.com/photo-1574158622682-e40e69881006?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1400&q=80',
+  },
+  {
+    name: 'Raisins',
+    age: 4,
+    enjoys: 'being queen of the dogs',
+    profile_pic:
+      'https://images.unsplash.com/photo-1500259571355-332da5cb07aa?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=934&q=80',
+  },
+  {
+    name: 'Toast',
```

Comment:
Because we have two sets of mock data right now (one in Rails and one in React) it might be more helpful to distinguish the two sets with different data so that it's very clear from looking at it, where the data is coming from

Reply:
okay we changed the data being passed to be different than the mockCats in the front end